x-pack/legacy/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -15,7 +15,18 @@ import {
   dataAnalyticsExplainSchema,
 } from '../new_platform/data_analytics_schema';
 
+/**
+ * Routes for the data frame analytics
+ */
 export function dataFrameAnalyticsRoutes({ xpackMainPlugin, router }: RouteInitialization) {
+  /**
+   * @api {get} /api/ml/data_frame/analytics Gets analytics data
```

Comment:
I think it's worth adding `@apiDescription` tags in for all the endpoints, even if they just contain some very basic information initially. For example `Returns the list of data frame analytics jobs.`

Reply:
https://github.com/elastic/kibana/pull/56006/commits/fc9297f3498431f41f91440b5c9ffa664401a3c5